stable/kong/README.md

```diff
@@ -48,43 +48,43 @@ chart and deletes the release.
-| proxy.nodePort                    | Node port when service type is `NodePort`                              |                       |
-| proxy.type                        | k8s service type. Options: NodePort, ClusterIP, LoadBalancer           | `NodePort`            |
-| proxy.loadBalancerIP              | To reuse an existing ingress static IP for the admin service           |                       |
-| proxy.ingress.enabled             | Enable ingress resource creation (works with proxy.type=ClusterIP)     | `false`               |
-| proxy.ingress.tls                 | Name of secret resource, containing TLS secret                         |                       |
-| proxy.ingress.hosts               | List of ingress hosts.                                                 | `[]`                  |
-| proxy.ingress.path                | Ingress path.                                                          | `/`                   |
-| proxy.ingress.annotations         | Ingress annotations. See documentation for your ingress controller for details | `{}`          |
-| env                               | Additional [Kong configurations](https://getkong.org/docs/latest/configuration/)               |
-| runMigrations                     | Run Kong migrations job                                                | `true`                |
-| readinessProbe                    | Kong readiness probe                                                   |                       |
-| livenessProbe                     | Kong liveness probe                                                    |                       |
-| affinity                          | Node/pod affinities                                                    |                       |
-| nodeSelector                      | Node labels for pod assignment                                         | `{}`                  |
-| podAnnotations                    | Annotations to add to each pod                                         | `{}`                  |
-| resources                         | Pod resource requests & limits                                         | `{}`                  |
-| tolerations                       | List of node taints to tolerate                                        | `[]`                  |
+| Parameter                      | Description                                                                      | Default             |
+| ------------------------------ | -------------------------------------------------------------------------------- | ------------------- |
+| image.repository               | Kong image                                                                       | `kong`              |
+| image.tag                      | Kong image version                                                               | `0.14.0`            |
```

Comment:
Can we keep this to 0.14.1, I think this could came in here due to a rebase/merge from master.